layouts/partials/page/list.html

```diff
@@ -1,4 +1,21 @@
-{{.Content}}
+{{ $roles := .Params.roles | default "All Roles" }}
+{{ $uid := .File.UniqueID }}
+{{ $pageLink := .Page.RelPermalink }}
+{{ $articleId := .Params.id | default .File.Path }}
+{{ $slug := (partial "common/slug" .) }}
+{{ $disableArticleBar := (eq (len .Content) 0) }}
+
+<div class="page index" data-roles="{{ $roles }}" id="{{ $uid }}" data-link="{{ $pageLink }}" permalink="{{ .Page.RelPermalink }}" data-disable-article-bar="{{ $disableArticleBar }}">
```

Comment:
This div is like the article/root.html but is missing the `<div class="article-title article-actions" data-align="center-left">` which renders the edit/delete icon.

This is because of styling as I did not want to rewrite the entire template and styling.

When rendering the WYSIWYG, it will use the data from this div.